corporate/tests/test_support_views.py

```diff
@@ -558,18 +557,26 @@ def test_discount_support_actions_when_upgrade_scheduled(self) -> None:
         )
         result = self.client_post(
             "/activity/remote/support",
-            {"remote_realm_id": f"{remote_realm.id}", "discount": "50"},
+            {
+                "remote_realm_id": f"{remote_realm.id}",
+                "monthly_discounted_price": "50",
+                "annual_discounted_price": "500",
+            },
         )
         self.assert_in_success_response(
-            ["Discount for realm-name-4 changed to 50% from 0%."], result
+            [
+                "Monthly price for realm-name-4 changed to 50 from 0. Annual price changed to 500 from 0."
```

Comment:
The "from 0" feels odd here; probably that suggests the success message code should be saying "from the default price" for this case.